usaon_vta_survey/templates/response/applications.html

```diff
@@ -54,31 +76,33 @@ <h3>Applications</h3>
     {% endif %}
   </div>
 
-  <h3>Add application</h3>
-  <form method="POST">
-    {{render_form(form, button_map={'submit_button': 'primary'}) }}
+  {% if not current_user.role_id=='analyst' %}
```

Comment:
Also, we should be doing the role check at the route level too; someone could manufacture a request to create a new application with `curl` instead of using our GUI, and in that case the only place to catch the access violation is in the route.

Reply:
I think you're totally right on this. I am working on it now